packages/jest-haste-map/src/index.ts

```diff
@@ -1149,6 +1149,9 @@ function copyMap<K, V>(input: Map<K, V>): Map<K, V> {
 type IJestHasteMap = HasteMapStatic & {
   create(options: Options): Promise<IHasteMap>;
   getStatic(config: Config.ProjectConfig): HasteMapStatic;
+  default: new (options: Options) => IHasteMap;
 };
+
+// @ts-expect-error(property `default is missing in  HasteMap class`)
```

Comment:
Can we cast instead or something? Adding type safety followed by a `@ts-ignore` seems unfortunate 😅